tasks/publish.yaml

```diff
@@ -57,22 +55,24 @@ tasks:
       - description: "Publish the bundles"
         cmd: |
           set -e
-          uds publish bundles/k3d-standard/uds-bundle-k3d-core-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
-          uds publish bundles/k3d-istio/uds-bundle-k3d-core-istio-amd64-${VERSION}.tar.zst ${TARGET_REPO}/bundles/${FLAVOR} --no-progress
 
-          # dont publish arm64 for registry1 since IB images are only amd64
+
+          # dont publish dev & demo bundles for registry1 flavor
           if [ "${FLAVOR}" != "registry1" ]; then
```

Comment:
nit: can drop the `if [ "${FLAVOR}" != "registry1" ]; then` conditional wrapper here since this isn't getting flavor passed in anymore I think?